source/kaleidic/lubeck.d

```diff
@@ -334,6 +334,16 @@ unittest
     assert(equal!((a, b) => a.approxEqual(b, 1e-10L, 1e-10L))(a.as!cdouble.inv.as!double, ans));
 }
 
+unittest
+{
+    import mir.algorithm.iteration: all;
+
+    auto identity = slice!double([4, 4], 0);
+	identity.diagonal[] = 1;
+
+    assert(identity.inv.all!"a > 0");
```

Comment:
It should check sign bits "a > 0", checks that all values are positive. "a >= 0" would pass for negative zeros.

Reply:
> btw, from where the tests are come from?

just from me.